live/mirage/routes/post-answers.js

```diff
@@ -17,11 +15,14 @@ import refQrocmAnswer from '../data/answers/ref-qrocm-answer';
 
 export default function (schema, request) {
 
+  /* eslint-disable */
```

Comment:
mouais... ce qui est beaucoup plus facile et efficace pour débugger une application c'est de cliquer sur "Lancer l'application en mode debug" et de mettre des breakpoints. Je n'ai pas trouvé de plugin pour Sublime Text. Par contre, depuis quelques mois il est possible de [débugger une application Node.js dans Chrome](https://medium.com/@paul_irish/debugging-node-js-nightlies-with-chrome-devtools-7c4a1b95ae27#.y68ar7ue4) ou d'[utiliser directement la console Chrome Dev Tools (sans Chrome)](https://mattdesl.svbtle.com/debugging-nodejs-in-chrome-devtools).